website/docs/r/stack_hci_deployment_setting.html.markdown

```diff
@@ -0,0 +1,585 @@
+
+---
+
+A `qos_policy_override` block supports the following:
+
+* `bandwidth_percentage_smb` - (Optional) Specifies the bandwidth allocation in % for the storage traffic. This parameter should only be modified based on your OEM guidance. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `priority_value8021_action_cluster` - (Optional) Specifies the Cluster traffic priority. This parameter should only be modified based on your OEM guidance. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `priority_value8021_action_smb` - (Optional) Specifies the Priority Flow Control where Data Center Bridging (DCB) is used. This parameter should only be modified based on your OEM guidance. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+---
+
+A `scale_unit` block supports the following:
+
+* `adou_path` - (Required) Specify the full name of the Active Directory Organizational Unit container object prepared for the deployment, including the domain components. For example:`OU=HCI01,DC=contoso,DC=com`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `cluster` - (Required) A `cluster` block as defined above. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `domain_fqdn` - (Required) Specifies the FQDN to deploy cluster. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `host_network` - (Required) A `host_network` block as defined above. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `infrastructure_network` - (Required) One or more `infrastructure_network` blocks as defined above. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `naming_prefix` - (Required) Specifies the naming prefix to deploy cluster. It must be 1-8 characters long and contain only letters, numbers and hyphens Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `optional_service` - (Required) A `optional_service` block as defined above. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `physical_node` - (Required) One or more `physical_node` blocks as defined above. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `secrets_location` - (Required) The URI to the keyvault or secret store. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `security_setting` - (Required) A `security_setting` block as defined below. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `storage` - (Required) A `storage` block as defined below. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `observability` - (Required) A `observability` block as defined above. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+---
+
+A `security_setting` block supports the following:
```

Comment:
please confirm the default values for all the sub properties

Reply:
The `security_setting` block is flattened and all sub-properties has set default values.